setup.py

```diff
@@ -56,7 +56,7 @@
     data_files=[(
         (BASE_DIR, ['data/nssm_original.exe'])
     )],
-    install_requires=['indy-plenum-dev==1.4.419',
+    install_requires=['indy-plenum-dev==1.4.420g',
```

Comment:
A typo `g`?

Reply:
Yes, i realised but was waiting for another release. Thanks for catching that